src/protobuf/buffer.cr

```diff
@@ -132,6 +155,14 @@ module Protobuf
       {tag, wire}
     end
 
+    def peek_info
+      n = peek_uint64
+      return {nil, nil} if n.nil?
+      tag = n >> 3
+      wire = (n & 0x7)
+
+      {tag, wire}
```

Comment:
Is `wire` used at all? The [only usage of `peek_info` I can find](https://github.com/jeromegn/protobuf.cr/pull/32/files#diff-3d29610fc8c1d179585a8d856bda0523R79) discards it. Am I overlooking something?

Reply:
Oh wait, is it to maintain parity with `read_info`?

Reply:
You are correct.  `wire` wasn't used at all for my peek implementation, but needs to exist with how the interface works to maintain parity with `read_info`.